common/elasticsearch/mapping.go

```diff
@@ -121,6 +122,10 @@ func customMetricTypeSchema(typeName string, customSchema string) string {
             }
           }
         },
+		"ClusterName": {
```

Comment:
please fix indentation. and use `cluster_name` (to keep a common case for the tags)